### PR TITLE
Adding draft project proposal of: Ventilator Knowledge Database

### PR DESCRIPTION
**Pull request by VictorMirandaEE**

### CODE REVIEW

**Comment by VictorMirandaEE** [comment]
Project draft is here: https://github.com/VictorMirandaEE/resources/blob/PROJECT-ventilator_knowledge_database/software/proposals/ventilator_knowledge_database.md

File: software/proposals/ventilator_knowledge_database.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## 📃 Idea in brief: Question to be resolved and why
+Web and mobile app to track the status of ventilator equipment in the public health care system and provide access to technical resources to help their maintenance.
+
+Main functionalities:
+- Ventilator Tracking Dashboard
+- Ventilator Technical Resources (HowTo guides, list of manufacturers and specialists, etc.)
+
+### 💥 Problem
+Mechanical ventilation may be needed in the treatment of the more severe respiratory distress cases of the COVID-19 disease (https://openwho.org/courses/introduction-to-ncov).
+One major concern is the availability of proper patient care equipment in the worst-case scenario of the COVID-19 outbreak, especially in the countries with low Universal Health Coverage (UHC) index (https://www.who.int/news-room/fact-sheets/detail/universal-health-coverage-(uhc)).
```
Comment (2001kraft) [inline software/proposals/ventilator_knowledge_database.md]:
```suggestion
One major concern is the availability of proper patient care equipment in the worst-case scenario of the COVID-19 outbreak, especially in the countries with low Universal Health Coverage (UHC) index (https://www.who.int/news-room/fact-sheets/detail/universal-health-coverage-(uhc)).
```
Please outline these in "Known Issues"

File: software/proposals/ventilator_knowledge_database.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Therefore, it is important to track the availability of this kind of equipment in a given geographic region, including their operational conditions.
+If maintenance is needed, volunteers will need access to verified technical resources such as training and HowTo/DIY guides provided/reviewed by specialists to perform proper repair and safe deployment.
+
+### 👨‍🔬Hypothesis
+
+### 🤔 Assumptions
+
+## 💻Implementation Suggestion
```
Comment (2001kraft) [inline software/proposals/ventilator_knowledge_database.md:38]: Please fill out this category

File: software/proposals/ventilator_knowledge_database.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Design
+(Screenshot/drawing/link)
+
+### Data/Experiments
+(Detail all the things about the experiment/trial setting especially any data requirements)
+
+V2 (..repeat the process)
+
+## 💪 Impact of Potential Solution
```
Comment (2001kraft) [inline software/proposals/ventilator_knowledge_database.md:74]: Please state impact

File: software/proposals/ventilator_knowledge_database.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## 🙌 Criteria for Success
```
Comment (2001kraft) [inline software/proposals/ventilator_knowledge_database.md:77]: Please outline the KPIs

**Review by 2001kraft — CHANGES_REQUESTED** [review]
Please finish filling out the template. Otherwise, we will not be able to adequately assess your proposal.

**Review by 2001kraft — APPROVED** [review]

**Comment by VictorMirandaEE** [comment]
Review done. I did my best due to the timing constraints.

**Pull request merged**